### DISCUSSION TITLE
Welcome to MultiSEngine Discussions!

### DISCUSSION THREAD

**Discussion by Megghy** [Megghy/MultiSEngine#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Edgel7RedHot** [comment]
hello, i have a problem here. I don't know where to install MultiSEngine and how to configure it so that it works in conjunction with Tshock

**Reply by Megghy** [reply]
Unzip it anywhere you want, start it up, refer to the existing items in the configuration file and change it to the address of your server, and access it through the port in the configuration file.
